app/app.go

```diff
@@ -57,7 +57,7 @@ func addGlobalFlags(app *cli.App) {
 			Name:     "config, c",
 			Usage:    "Load configuration from `FILE`",
 			EnvVar:   "KORP_GLOBAL_CONFIG",
-			FilePath: "~/.korp/config",
+			FilePath: "./config",
```

Comment:
I don't think the current directory is a good location to read the config from. The user would have to write a config for each directory which they want to work in.

Reply:
Sure, it was just for make easier testing.